### ISSUE TITLE
Return relevant fields for push to card in unauthenticated card tokenization

### ISSUE THREAD

**Issue by matin** [balanced/balanced-api#615]
Currently, only the `href` and `id` are returned when tokenizing without a secret key.

The following should also be returned on the Card:
1. `can_credit`
2. `type`
3. `category`
4. `bank_name`

**Comment by remear** [comment]
@matin All unauthenticated tokenizations return ``href`` and ``id`` by design in API 1.1.

```json
{
    "cards": [
        {
            "href": "/cards/CC6jYBXR91ujWSkvnmPUmRy7",
            "id": "CC6jYBXR91ujWSkvnmPUmRy7",
            "links": {}
        }
    ],
    "links": {},
    "status_code": 201
}
```

```json
{
    "bank_accounts": [
        {
            "href": "/bank_accounts/BA6rJZeYQYCWdz9sWNXQFOET",
            "id": "BA6rJZeYQYCWdz9sWNXQFOET",
            "links": {}
        }
    ],
    "links": {},
    "status_code": 201
}
```

Additional fields are returned after claiming the funding instrument to the marketplace. Why should these other fields be returned on tokenization (and only for a card?)?

**Comment by matin** [comment]
@remear it allows the customer to manage the logic client side. It's a similar reason to how balanced.js can check card number and routing number validity.

**Comment by matin** [comment]
@msherry you're right that it makes sense to revisit that design decision. There are other ways to detect and block abuse.

@remear what caused you to visit this issue today?

**Comment by mahmoudimus** [comment]
lawl @msherry your'e reading too many academic papers (c.f.) :D

**Comment by remear** [comment]
@matin, pruning issues.
